### ISSUE TITLE
MPS (Metal Performance Shaders) support for training

### ISSUE THREAD

**Issue by ehsanfathi77** [ourownstory/neural_prophet#1572]
When I train a model on my Mac I get this warning:

MPS available but not used. Set `accelerator` and `devices` using `Trainer(accelerator='mps', devices=1)`.

Is there a way to take advantage of MPS to accelerate the training?

**Comment by MaiBe-ctrl** [comment]
@ehsanfathi77 Thanks for reporting this issue. Could you provide the exact logs that you are getting and the macOS version of your machine please ?

**Comment by MaiBe-ctrl** [comment]
We have thoroughly investigated this issue, and we were unable to find or reproduce the reported problem. As a result, we will be closing this issue for now. If the issue persists, please feel free to reopen it.
